docs/internals/contributing/writing-code/javascript.txt

```diff
@@ -30,10 +30,8 @@ Code style
   ``$(selector).click(func)``. This makes it easier for projects to extend
   Django's default behavior with JavaScript.
 
-.. _javascript-patches:
-
-JavaScript patches
-==================
+jQuery and performance
```

Comment:
Hi!

I think not everything is jQuery in the admin, I remember did a change to delete/reeplace jQuery code, so I wouldn't emphasize on it when there is also vanilla javascript.

And regarding the word "performance" I don't see relation with the documentation below.

So I would recommend apply the same as you did for: Optimization patches -> Optimization changes

jQuery and performance -> JavaScript changes

Reply:
I was a bit unsure how to rename the title, considering that the following section consists of litterally these two sentences, nothing else (emphasis mine):

> Django's admin system leverages **the jQuery framework** to increase the
> capabilities of the admin interface. In conjunction, there is an emphasis on
> admin **JavaScript performance** and minimizing overall admin media file size.

I don't mind changing the title of this section to "JavaScript changes" if you think that's better, but I don't really see how it explains the content of the section better. 🤔

Reply:
`JavaScript in the Django admin` maybe?